test/unit/much-result_tests.rb

```diff
@@ -170,6 +170,8 @@ class InitTests < UnitTests
     should have_imeths :get_for_all_results
     should have_imeths :get_for_all_success_results, :get_for_all_failure_results
     should have_imeths :to_much_result
+    should have_imeths :attributes
+    should have_imeths :attribute_names
```

Comment:
@pdobb oops I "Previewed" this one and never hit "Submit":

Please juxtapose after the `should have_imeths :description, :backtrace, :set` line above. Also collapse to a single line since it will fit and the methods are related:
```suggestion
    should have_imeths :attributes, :attribute_names
```